tests/integration/targets/mso_schema_template/tasks/main.yml

```diff
@@ -38,18 +38,22 @@
     schema: '{{ item }}'
     state: absent
   loop:
+  - '{{ mso_schema | default("ansible_test") }}_3'
   - '{{ mso_schema | default("ansible_test") }}_2'
   - '{{ mso_schema | default("ansible_test") }}'
 
 - name: Ensure tenant ansible_test exists
   mso_tenant: 
     <<: *mso_info
-    tenant: ansible_test
+    tenant: '{{ item }}'
     users:
       - '{{ mso_username }}'
     sites:
     - '{{ mso_site | default("ansible_test") }}'
     state: present
+  loop:
+  - 'ansible_test'
+  - 'ansible_test_2'
```

Comment:
You add a tenant here and a schema after but does not change the cleanup section of the next module so I am afraid one module will fail at the next execution of this one.

Reply:
I added a cleanup section for the extra tenant and schema I included here. That should take care of this right?

Reply:
You might need to add those at the top of the next module also in case this module fails so the future modules don't have any issues.

Reply:
okay